test/validation/api/init/init_main.c

```diff
@@ -5,11 +5,14 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+#define _GNU_SOURCE
```

Comment:
Why this needs to be added?

Reply:
It doesn't. Left behind by mistake. Removed in v4.